www/js/survey/enketo/enketoHelper.ts

```diff
@@ -14,14 +17,137 @@ export type SurveyOptions = {
   dataKey?: string;
 };
 
+type EnketoAnswerData = {
+  label: string; //display label (this value is use for displaying on the button)
+  ts: string; //the timestamp at which the survey was filled out (in seconds)
+  fmt_time: string; //the formatted timestamp at which the survey was filled out
+  name: string; //survey name
+  version: string; //survey version
+  xmlResponse: string; //survey answer XML string
+  jsonDocResponse: string; //survey answer JSON object
+}
+
+type EnketoAnswer = {
+  data: EnketoAnswerData; //answer data
+  metadata: any;
+}
+
+type EnketoSurveyConfig = {
+  [surveyName:string]: {
+    formPath: string
+    labelTemplate: {[lang: string] : string};
+    labelVars: {[activity: string]: {[key: string]: string, type:string}},
+    version: number;
+    compatibleWith: number;
+  }
+}
+
+/** @type {EnketoSurveyConfig} _config */
+//TODO find a more appropriate way to store this
+let _config: EnketoSurveyConfig;
```

Comment:
Actually, I don't think there's anything wrong with this. It's a pattern I've been using in the other rewrites.

Since this file is a module (ie it uses import/export syntax), `_config` is scoped to this file; it is not polluting the global scope.

I do usually put the variable (`_config` in this case) right above the function that uses it.

Reply:
Ok, good to know! I didn't see a great way around it, since I knew `useState` wasn't the answer. I will move it though so the use case is clearer

Reply:
Moved and removed the comment in [6ef2725](https://github.com/e-mission/e-mission-phone/pull/1063/commits/6ef2725b21a5bd613752904b93af346fd543702a)